libs/postgres_ffi/wal_generate/src/lib.rs

```diff
@@ -250,6 +253,84 @@ pub fn generate_simple(client: &mut impl postgres::GenericClient) -> Result<PgLs
     })
 }
 
+pub fn generate_last_wal_record_xlog_switch(
```

Comment:
Was there there a specific bug targeted by this test? `last_wal_record_xlog_switch` seems to be enough for #1190.